modules/apps/commerce/commerce-catalog-web/src/main/java/com/liferay/commerce/catalog/web/internal/frontend/data/set/view/table/CommerceCatalogTableFDSView.java

```diff
@@ -44,7 +44,6 @@ public FDSTableSchema getFDSTableSchema(Locale locale) {
 			"name", "name",
 			fdsTableSchemaField -> {
 				fdsTableSchemaField.setContentRenderer("actionLink");
-				fdsTableSchemaField.setExpand(true);
 			}
```

Comment:
Since there is only one item in lambda, we can simplify this to 
```java
fdsTableSchemaField -> fdsTableSchemaField.setContentRenderer("actionLink")
```

We have a source formatter script that would catch this. You can run `formatSource` from the directory of the module. You will get a failure with 
```
java.lang.Exception: Found 1 formatting issues:
1: Simplify lambda statement with single statement, see https://github.com/liferay/liferay-portal/blob/master/modules/util/source-formatter/src/main/resources/documentation/check/lambda_check.markdown: /home/marko/git/liferay-master/portal/modules/apps/commerce/commerce-catalog-web/src/main/java/com/liferay/commerce/catalog/web/internal/frontend/data/set/view/table/CommerceCatalogTableFDSView.java 45 (Checkstyle:LambdaCheck)
```

Reply:
> $ git pull upstream master
$ git checkout -b LPS-168867
$ git commit -a -m "LPS-168867 ....."
$ git push origin LPS-168867

This would work fine if you start with `master` as your current branch, and you don't have `LPS-168867`. When you pull `master`, there is no difference if you merge or rebase.

But if you run this with `LPS-168867` as your current branch, with few commits on top and behind master, then it becomes an issue that you are merging.

Reply:
>I think one git pull upstream master --rebase would fix this

Or maybe not, you are probably getting the same errors as me :thinking: 

You can checkout a new branch from master, cherry-pick PR commits on it, and force-push. That will surely work.

Reply:
Because of these kinds of shenanigans with git I like to have a graphical tool, like smartgit :sweat_smile:

Reply:
> 
> Or maybe not, you are probably getting the same errors as me thinking
> 
Yes. I run `git pull upstream master -rebase` and have the same issues with the rebase:
```
liferay-portal (master) $ git pull upstream master --rebase
remote: Enumerating objects: 2801, done.
remote: Counting objects: 100% (2800/2800), done.
remote: Compressing objects: 100% (765/765), done.
remote: Total 2270 (delta 1095), reused 1836 (delta 733), pack-reused 0
Recibiendo objetos: 100% (2270/2270), 19.92 MiB | 1.85 MiB/s, listo.
Resolviendo deltas: 100% (1095/1095), completado con 271 objetos locales.
Desde github.com:liferay/liferay-portal
 * branch                      master     -> FETCH_HEAD
   6bf7e58d233a..7f3d45d92df2  master     -> upstream/master
Auto-fusionando modules/dxp/apps/osb/osb-faro/.gitrepo
CONFLICTO (contenido): Conflicto de fusión en modules/dxp/apps/osb/osb-faro/.gitrepo
error: no se pudo aplicar 4d62dead0bc7... subrepo:ignore Update 'modules/dxp/apps/osb/osb-faro/.gitrepo'.
ayuda: Resolve all conflicts manually, mark them as resolved with
ayuda: "git add/rm <conflicted_files>", then run "git rebase --continue".
ayuda: You can instead skip this commit: run "git rebase --skip".
ayuda: To abort and get back to the state before "git rebase", run "git rebase --abort".
No se pudo aplicar 4d62dead0bc7... subrepo:ignore Update 'modules/dxp/apps/osb/osb-faro/.gitrepo'.
```